Code/Tools/FBuild/FBuildCore/BFF/Functions/FunctionCompiler.cpp

```diff
@@ -41,19 +41,7 @@ FunctionCompiler::FunctionCompiler()
 //------------------------------------------------------------------------------
 /*virtual*/ bool FunctionCompiler::Commit( NodeGraph & nodeGraph, const BFFIterator & funcStartIter ) const
 {
-    AStackString<> name;
-    if ( GetNameForNode( nodeGraph, funcStartIter, CompilerNode::GetReflectionInfoS(), name ) == false )
-    {
-        return false;
-    }
-
-    if ( nodeGraph.FindNode( name ) )
```

Comment:
If compilers use the m_AliasForFunction name does it still need to be checked for uniqueness?